doc/man1/flux-dmesg.rst

```diff
@@ -34,6 +34,17 @@ OPTIONS
 **-n, --new**
    Follow only new log entries.
 
+**-H, --human**
+   Display human-readable output. See also **--color** and **--delta**.
+
+**-d, --delta**
+   With **--human**, display the time delta between messages instead
+   of a relative offset since the last absolute timestamp.
+
+**-L, --color**\ *[=WHEN]*
+   Colorize output. The optional argument *WHEN* can be *auto*, *never*,
+   or *always*. If *WHEN* is omitted, it defaults to *always*. The default
+   value if **--color** is not used is *auto*.
```

Comment:
value if -> value of?

Reply:
No, "The value of --color is not used is _auto_" won't make sense either. Obviously the wording is confusing though, so how about simply "The default value is `auto`"?